README.md

```diff
@@ -15,22 +15,29 @@ please ask in the [project forums](https://forum.openbikesensor.org).
 
 This repository contains a lot of files for different options
 
```

Comment:
Ordered alphabetically because that's how the folders display on github and on the user machine (likely)